launchable/utils/launchable_client.py

```diff
@@ -39,7 +40,7 @@ def request(
         timeout: Tuple[int, int] = (5, 60),
         compress: bool = False,
         additional_headers: Optional[Dict] = None,
-    ):
+    ) -> requests.Response:
```

Comment:
Technically this is not correct as we can sometimes return DryRunResponse, but this type signature makes auto-completion better. What's the right way to do this?